master/internal/db/postgres_users.go

```diff
@@ -234,10 +234,10 @@ func deleteAgentUserGroup(tx *sqlx.Tx, userID model.UserID) error {
 }
 
 // AddUser creates a new user.
-func (db *PgDB) AddUser(user *model.User, ug *model.AgentUserGroup) error {
+func (db *PgDB) AddUser(user *model.User, ug *model.AgentUserGroup) (model.UserID, error) {
```

Comment:
perhaps return a nil pointer here instead of 0 value?

```suggestion
func (db *PgDB) AddUser(user *model.User, ug *model.AgentUserGroup) (*model.UserID, error) {
```

Reply:
do you think this is a blocker? tried using the pointer and nil values and starting running into a bunch of different errors that led to other errors... happy to address if it's important but also want to try to get this fix in ASAP.

Reply:
What kind of errors do you see?